File: solucoes/2021/nivel jr/fase 2/js/pesquisa.js
var N, E, A, G;
scanf("%d", "N");
var estados = [];

for (var i = 0; i < N; i++) {
  scanf("%d%d%d", "E", "A", "G");
  var vantajoso = G * 0.7;
  if (A <= vantajoso) {
    estados.push(E);
  }
}
if (estados == "") {
  printf("%s", "*");
} else {
  for (var i = 0; i < estados.length; i++) {
    printf("%s", estados[i]);
  }
}

File: solucoes/2021/nivel jr/fase 2/py/pesquisa.py
N = int(input())
estados = []

for i in range(N):
    E, A, G = list(map(str, input().split()))
    vantajoso = float(G) * 0.70
    if float(A) <= vantajoso:
        estados.append(E)

if not estados:
    print('*')
else:
    for i in range(len(estados)):
        print(estados[i])